vector/src/main/res/values/colors.xml

```diff
@@ -19,6 +19,8 @@
     <color name="tchap_search_bar">#90dee5f1</color>
     <color name="tchap_search_bar_text">#7d7d7d</color>
     <color name="tchap_action_bar_text_color">#ffffff</color>
+    <color name="tchap_text_color_black">#000000</color>
+    <color name="tchap_text_color_grey">#aeaeae</color>
```

Comment:
You should not put the color in the name here
tchap_text_color_black -> tchap_primary_text_color
tchap_text_color_grey -> tchap_secondary_text_color